### PR TITLE
Update exitId from changes made in plasma-contracts

### LINKED ISSUE #1507: ExitId: implement new exitId in elixir-omg
https://github.com/omisego/plasma-contracts/issues/622

### PR DESCRIPTION
**Pull request by souradeep-das**
closes #1507 

## Overview

Modify the size of exitId in accordance with the changes to plasma-contracts

### CODE REVIEW

**Comment by souradeep-das** [comment]
Changed the exitId's in some tests to exitId's with the new format to avoid any confusion

**Comment by boolafish** [comment]
@souradeep-das you would probably need to go to the docker repo (and different org 😅) to update the plasma-contract sha and generate a snapshot with a branch there.

After that, you can update the snapshot file here to try to make all the tests passes with new exitId.

https://github.com/omisego-images/docker-elixir-omg

**Comment by boolafish** [comment]
why are you closing this? @unnawut

**Comment by unnawut** [comment]
I'm wondering the same....

**Comment by unnawut** [comment]
Ah it's the branch 🤦

**Comment by boolafish** [comment]
oh...you're cleaning the branches?

**Comment by unnawut** [comment]
Yeah right now the `v*` branches are used for releases. Can we re-target this PR to a branch with some other names? Similar to how you did with payment v2 poc?

**Comment by boolafish** [comment]
it is supposed to be everything on v2 network. So...I don't what what a better branch name should be 😅 If you got suggestion we can use that : )

**Comment by unnawut** [comment]
Ahh I see. How about `plasma-v2`?

If okay, I'll go ahead fix this PR. Sorry for the confusion 🙇

**Comment by boolafish** [comment]
probably I might prefer some stupid name like `master-v2` then 😛 

ah....seems like "master" is going to be banned word 😅: https://www.facebook.com/241806149201604/posts/3024938127555045/?d=n

**Comment by boolafish** [comment]
Anyway @souradeep-das decide a name you like hahah

**Comment by unnawut** [comment]
Revived the PR for now but feel free to name the target branch that works best!

**Comment by souradeep-das** [comment]
haha, we could probably use `master-v2` for now 😛
Should we also do this to `plasma-contracts` and `ex_plasma` to maintain consistency?

**Comment by souradeep-das** [comment]
For reference:- 
Using plasma-contracts snapshot from https://github.com/omisego-images/docker-elixir-omg/tree/new_exitId_format
Using ex_plasma branch `v2.0.0-backport` https://github.com/omgnetwork/ex_plasma/tree/v2.0.0-backport

**Comment by souradeep-das** [comment]
For reference:-
Using plasma-contracts snapshot from https://github.com/omisego-images/docker-elixir-omg/tree/new_exitId_format

Using ex_plasma branch `v2.0.0-backport` https://github.com/omgnetwork/ex_plasma/tree/v2.0.0-backport and branch `master-v2` https://github.com/omgnetwork/ex_plasma/tree/master-v2

File: apps/omg/mix.exs [hunk 1]
```diff
@@ -32,7 +32,7 @@ defmodule OMG.MixProject do
 
   defp deps() do
     [
-      {:ex_plasma, git: "https://github.com/omisego/ex_plasma.git", ref: "83db9a0f5a441f7e59ae23f8c64658e2c387f8a5"},
+      {:ex_plasma, git: "https://github.com/omisego/ex_plasma.git", ref: "bd42028341d9232e11a084dbc5adac08c8bf5672"},
```
Comment (achiurizo) [inline apps/omg/mix.exs:35]:
```suggestion
      {:ex_plasma, git: "https://github.com/omgnetwork/ex_plasma.git", ref: "bd42028341d9232e11a084dbc5adac08c8bf5672"},
```

**Pull request merged**